net/respondd/src/respondd.c

```diff
@@ -649,10 +645,7 @@ int main(int argc, char **argv) {
 			break;
 
 		case 'd':
-			if (chdir(optarg)) {
```

Comment:
Now I open the cache file for every provider that is loaded, thus also in every directory a request type is defined in, and the last one has precedence because earlier values are simply overridden if the file exists.